duckduckgo/iOS
# Translation for accessibility labels

Task/Issue URL: https://app.asana.com/0/414235014887631/1202854620659929/f
Tech Design URL:
CC: @mallexxx 

**Description:**
Add missing translation to localization strings

**Steps to test this PR:**

Just checking the strings file should be good, but if you're feeling inspired you can:

1. Change your device to a non-english but supported language
2. Turn on voice over on accessibility
3. Check Auto fill show/hide button
4. Check the auto-complete suggestion accessibility labels

**Device Testing**:




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
